cvat/apps/engine/migrations/0087_job_last_export_date_project_last_export_date_and_more.py

```diff
@@ -0,0 +1,28 @@
+# Generated by Django 4.2.15 on 2024-12-09 16:51
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("engine", "0086_profile_has_analytics_access"),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name="job",
+            name="last_export_date",
+            field=models.DateTimeField(null=True),
+        ),
+        migrations.AddField(
+            model_name="project",
+            name="last_export_date",
+            field=models.DateTimeField(null=True),
+        ),
+        migrations.AddField(
+            model_name="task",
+            name="last_export_date",
+            field=models.DateTimeField(null=True),
+        ),
+    ]
```

Comment:
Should there be something like a one-off cleanup job to clean old exports? As a workaround, set last export date to creation date, otherwise objects will be skipped by the current cleanup condition.

Reply:
You are right, but I thought we'd run such a one-off cleanup job/script manually after deploying this patch. I don't think that we need to add a data migration here, since such a migration can take a lot of time. Instead, we could specify in the documentation how to clean up the outdated cache.

> As a workaround, set the last export date to the creation date

This patch tries to uniform the naming of files saved in the export cache (specifically backups) without backward compatibility. Since a file could have been stored in the cache for less than 1 day, I don't think that we need to support files with old name patterns. So, this workaround is not fully robust.